ipt_connect/IPTdev/func_bonus.py

```diff
@@ -0,0 +1,63 @@
+#Functions for calculating bonus points
+
+def distribute_bonus_points(points_list):
+	if len(points_list) == 3:
+		return distribute_bonus_points_3(points_list)
+	if len(points_list) == 4:
+		return distribute_bonus_points_4(points_list)
+	return [2.0, 1.0, 1.0, 0.0, 0.0]
```

Comment:
what is this for?
And, according to current rules, for a hypothetical 5-teams fight the distribution would be 2-1-1-1-0